tls/s2n_fingerprint.h

```diff
@@ -40,7 +40,7 @@ struct s2n_fingerprint_hash {
 S2N_RESULT s2n_fingerprint_hash_add_char(struct s2n_fingerprint_hash *hash, char c);
```

Comment:
Okay my last remaining annoying thought I had about this project, why don't the function signatures for `s2n_stuffer_hex_read` and `s2n_stuffer_read` match?
```
int s2n_stuffer_read(struct s2n_stuffer *stuffer, struct s2n_blob *out)
S2N_RESULT s2n_stuffer_read_hex(struct s2n_stuffer *bytes_out, const struct s2n_blob *hex_in)
```

Reply:
That's still on my to-do list! It's just a bigger change so I wasn't going to bundle it with these little ones.